### ISSUE TITLE
Gregg Review of all AAA SC - with Preliminary Recommendation

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag2ict#791]
## GV preliminary AAA Review (2022)
In 2022 when we first were setting up WCAG2ICT and deciding whether or not to include AAA provisions, I did a quick review of all of the AAA provisions to see if it was a reasonable amount of work.

### My PRELIMINARY results were:

> **_ALL  AAA provisions apply as written, and as described in Understanding WCAG 2.2 except  3 of them:_**

---
>#### 2.4.8 Location
>A mechanism is available to allow the purpose of each link to be identified from link text alone, except where the purpose of the link would be ambiguous to users in general.

Applies as written with Software or Electronic documents , and as described in Intent from Understanding Success Criterion 2.4.8, replacing “**_within a set of Web pages_**” with “**_in a set of non-web documents or software”_**.

---
>#### 2.5.6 Concurrent Input Mechanisms
>Web content does not restrict use of input modalities available on a platform except where the restriction is essential, required to ensure the security of the content, or required to respect user settings.

Applies directly as written, and as described in Intent from Understanding Success Criterion 2.5.6, replacing “**_Web content”_** with “**_non-web content or software_**”.

---
>#### 3.3.6 Error Prevention (All)
>For Web pages that require the user to submit information, at least one of the following is true:
>- Reversible: Submissions are reversible.
>- Checked: Data entered by the user is checked for input errors and the user is provided an opportunity to correct them.
>- Confirmed: A mechanism is available for reviewing, confirming, and correcting information before finalizing the submission.

Applies directly as written, and as described in Intent from Understanding Success Criterion 3.3.6 (also provided below), replacing “**_Web pages_**” with “**_non-web documents or software”_**.

---
---

**HOWEVER** these are preliminary BEFORE we did any of the other discussions so should be treated as such.   We are now ready for the full group to review. For this purpose, the AAA SC have all been given their own ISSUE for review comments.   

- Here is a [link to the original 2022 word doc ](https://github.com/w3c/wcag2ict/blob/GreggVan-patch-1/WCAG2ICT%20Gregg%20Vanderheiden%20Proposal%20for%20AAA%20provisions.docx) with **the full review all in one piece i**f this is easier for review 
- Below are the links to the issues for each AAA SC individually.   

All of those SC that have no comments on them - are listed first - followed by those that have comments.   This separation is provided to make it easy to identify all of those that fall into the "APPLY AS WRITTEN" category from those we want to make comments on.    This split will be updated weekly but if you make a comment and want to move something from the APPLY AS WRITTEN list below to the  COMMENTED LIST - please feel free to do so. 

---

### SC's  CURRENTLY WITH NO COMMENTS AGAINST  "APPLY AS WRITTEN" 

- [1.2.6: Audio Description (Prerecorded) (Level AAA)](https://github.com/w3c/wcag2ict/issues/532)
- [1.2.7 Extended Audio Description (Prerecorded) (Level AAA)](https://github.com/w3c/wcag2ict/issues/533)
- [1.2.8 Media Alternative (Prerecorded) (Level AAA)](https://github.com/w3c/wcag2ict/issues/534)
- [1.2.9 Audio-only (Live) (Level AAA)](https://github.com/w3c/wcag2ict/issues/535)
- [1.3.6 Identify Purpose (Level AAA)](https://github.com/w3c/wcag2ict/issues/538)[AAA](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [1.4.6 Contrast (Enhanced) (Level AAA)](https://github.com/w3c/wcag2ict/issues/539)
- [1.4.7 Low or No Background Audio (Level AAA)](https://github.com/w3c/wcag2ict/issues/540)
- [1.4.8 Visual Presentation (Level AAA)](https://github.com/w3c/wcag2ict/issues/541)
- [1.4.9 Images of Text (No Exception) (Level AAA)](https://github.com/w3c/wcag2ict/issues/542)
- [2.1.3 Keyboard (No Exception) (Level AAA)](https://github.com/w3c/wcag2ict/issues/543)
- [2.2.3 No Timing (Level AAA)](https://github.com/w3c/wcag2ict/issues/544)
- [2.2.4 Interruptions (Level AAA)](https://github.com/w3c/wcag2ict/issues/545)
- [2.2.5 Re-authenticating (Level AAA)](https://github.com/w3c/wcag2ict/issues/546)
- [2.2.6 Timeouts (Level AAA)](https://github.com/w3c/wcag2ict/issues/547)[AAA](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [2.3.2 Three Flashes (Level AAA)](https://github.com/w3c/wcag2ict/issues/548)
- [2.3.3 Animation from Interactions (Level AAA)](https://github.com/w3c/wcag2ict/issues/549)
- [2.4.8 Location (Level AAA)](https://github.com/w3c/wcag2ict/issues/550)[AAA](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [2.4.9 Link Purpose (Link Only) (Level AAA)](https://github.com/w3c/wcag2ict/issues/551)
- [2.4.10 Section Headings (Level AAA)](https://github.com/w3c/wcag2ict/issues/552)
- [2.4.12 Focus Not Obscured (Enhanced) (Level AAA)](https://github.com/w3c/wcag2ict/issues/553)
- [2.4.13 Focus Appearance (Level AAA)](https://github.com/w3c/wcag2ict/issues/554)
- [2.5.5 Target Size (Enhanced) (Level AAA)](https://github.com/w3c/wcag2ict/issues/555)[AAA](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [2.5.6 Concurrent Input Mechanisms (Level AAA)](https://github.com/w3c/wcag2ict/issues/556)
- [3.1.3 Unusual Words (Level AAA)](https://github.com/w3c/wcag2ict/issues/558)
- [3.1.4 Abbreviations (Level AAA)](https://github.com/w3c/wcag2ict/issues/559)
- [3.1.5 Reading Level (Level AAA)](https://github.com/w3c/wcag2ict/issues/560)
- [3.1.6 Pronunciation (Level AAA)](https://github.com/w3c/wcag2ict/issues/561)
- [3.2.5 Change on Request (Level AAA)](https://github.com/w3c/wcag2ict/issues/562)
- [3.3.5 Help (Level AAA)](https://github.com/w3c/wcag2ict/issues/563)[AAAAny AAA related requirement](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [3.3.9 Accessible Authentication (Enhanced) (Level AAA)](https://github.com/w3c/wcag2ict/issues/565)[AAAAny AAA related requirement](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)

### SC's WITH COMMENTS ON THEM
- [2.4.8 Location (Level AAA)](https://github.com/w3c/wcag2ict/issues/550)[AAA](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
- [2.5.6 Concurrent Input Mechanisms (Level AAA)](https://github.com/w3c/wcag2ict/issues/556)
- [3.3.6 Error Prevention (All) (Level AAA)](https://github.com/w3c/wcag2ict/issues/564)[AAAAny AAA related requirement](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue%20state%3Aopen%20label%3AAAA)
